edk2toolext/SBAT/PEObject.py

```diff
@@ -0,0 +1,123 @@
+import pefile
+import peutils
+import sys
+import json
+
+FILE_OS_STRINGS = {
```

Comment:
I had asked Garand to document some of where this came from.  I believe in a PR he has references to spec, etc.  

I think this file needs some documentation.